reactor-client/src/main/java/com/aerospike/client/reactor/AerospikeReactorClient.java

```diff
@@ -0,0 +1,467 @@
+package com.aerospike.client.reactor;
+
+import com.aerospike.client.*;
+import com.aerospike.client.async.EventLoops;
+import com.aerospike.client.listener.*;
+import com.aerospike.client.policy.*;
+import com.aerospike.client.query.KeyRecord;
+import com.aerospike.client.query.Statement;
+import com.aerospike.client.reactor.dto.KeyExists;
+import com.aerospike.client.reactor.dto.KeyObject;
+import com.aerospike.client.reactor.dto.KeysExists;
+import com.aerospike.client.reactor.dto.KeysRecords;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.util.List;
+
+/**
+ * Instantiate an <code>AerospikeReactorClient</code> object to access an Aerospike
+ * database cluster and perform database operations reactively.
+ * <p>
+ * This client is just wrapper over AerospikeClient async methods that provides Reactor interface
+ * <p>
+ * This client is thread-safe. One client instance should be used per cluster.
+ * Multiple threads should share this cluster instance.
+ * <p>
+ * Your application uses this class API to perform database operations such as
+ * writing and reading records, and selecting sets of records. Write operations
+ * include specialized functionality such as append/prepend and arithmetic
+ * addition.
+ * <p>
+ * Each record may have multiple bins, unless the Aerospike server nodes are
+ * configured as "single-bin". In "multi-bin" mode, partial records may be
+ * written or read by specifying the relevant subset of bins.
+ *
+ * @author Sergii Karpenko
+ */
+public class AerospikeReactorClient implements IAerospikeReactorClient{
+
+	private final AerospikeClient aerospikeClient;
+	private final EventLoops eventLoops;
+
+	public AerospikeReactorClient(AerospikeClient aerospikeClient, EventLoops eventLoops) {
+		this.aerospikeClient = aerospikeClient;
+		this.eventLoops = eventLoops;
```

Comment:
Could you please check how EventLoops is setup/closed in `com.aerospike.client.cluster.Cluster`?
Maybe we should reuse this code here?